recipe/meta.yaml

```diff
@@ -55,18 +51,15 @@ outputs:
         - numpy
         # exclude 8.3.0 and 8.3.1 specifically due to pytorch/vision#4146, python-pillow/Pillow#5571
         - pillow >=5.3.0,!=8.3.0,!=8.3.1
-        - pytorch {{ pytorch_version }}
-        - pytorch-gpu                           # [cuda_compiler_version != "None"]
-        - pytorch-cpu                           # [cuda_compiler_version == "None"]
+        - pytorch
+        - pytorch =*={{ torch_proc_type }}*
       run:
         - python
-        - pytorch {{ pytorch_version }}
-        - pytorch-gpu                            # [cuda_compiler_version != "None"]
-        - pytorch-cpu                            # [cuda_compiler_version == "None"]
+        - pytorch
+        - pytorch =*={{ torch_proc_type }}*
         - {{ pin_compatible('cudnn') }}          # [cuda_compiler_version != "None"]
         - {{ pin_compatible('numpy') }}
         - pillow >=5.3.0,!=8.3.0,!=8.3.1
-        - ffmpeg
```

Comment:
Presumably this is handled by a run-export? Any other reason for removing it?

Reply:
run exports it is.

Reply:
Can you elaborate how this was handled because I am not following the logic...

Reply:
https://conda-forge.org/docs/maintainer/pinning_deps.html?highlight=run_exports#specifying-run-exports

Reply:
Yes, but I don't see run_exports in the meta... I thought it would be there

Reply:
https://github.com/conda-forge/ffmpeg-feedstock/blob/master/recipe/meta.yaml#L21